### ISSUE TITLE
Check if glmnet dependency is needed

### ISSUE THREAD

**Issue by koldLight** [EuroMOMOnetwork/MOMO#2]
Hi,

I'm trying to update MOMO in my system with R 3.5.

It fails with the error:

```
ERROR: dependency ‘glmnet’ is not available for package ‘MOMO’
```

If I try to install glmnet manually, it fails because R >= 3.6 is needed (cran ref [here](http://lib.stat.cmu.edu/R/CRAN/)):

```
package ‘glmnet’ is not available (for R version 3.5.0) 
```

But [after checking the use of glmnet in the source code](https://github.com/EuroMOMOnetwork/MOMO/search?q=glmnet&unscoped_q=glmnet), it looks it's not needed anymore. If you can confirm this, could you consider removing it from the [`Imports` list](https://github.com/EuroMOMOnetwork/MOMO/blob/master/DESCRIPTION#L21)?

Thanks

**Comment by JensXII** [comment]
Hi Luz,

The dependency of glmnet has been removed. You were right, it’s not used.

Could you please try again, and let me know if it works now.

The EuroMOMO R.scripts are free for use and we are only happy, if anyone uses them. But, it would be nice, if you, please, would inform me, on nls@ssi.dk,  about why you are interested in using MOMO and on data from which country?
Best regards
  Jens

Jens Nielsen
Senior Scientist, Statistician, MSc PhD
Infectious Disease Epidemiology and Prevention, Statens Serum Institut

T(directe)  +45 3268 3965  | E nls@ssi.dk<mailto:nls@ssi.dk> | B 45/106 | ssi.dk
Artillerivej 5 | DK2300 Copenhagen S | Denmark

[Beskrivelse: Beskrivelse: Beskrivelse: Beskrivelse: Beskrivelse: Beskrivelse: Beskrivelse: Beskrivelse: Beskrivelse: Beskrivelse: Beskrivelse: Beskrivelse: Beskrivelse: SSI_logo_left_100px]

**Comment by koldLight** [comment]
Thanks, I close the issue as it was solved with #3 :)
